src/Engine/ProtoCore/Lang/CallSite.cs

```diff
@@ -142,7 +142,7 @@ internal void GetObjectData(SerializationInfo info, StreamingContext context, in
                     //Serialise the object
                     using (MemoryStream s = new MemoryStream())
                     {
-                        IFormatter formatter = new SoapFormatter();
+                        SoapFormatter formatter = new SoapFormatter();
```

Comment:
stopped using IFormatter to get around the error.

Reply:
How come this are failing now? Or are you already targeting net8?

Reply:
Oh..this should have been moved to the net8 PR.
I guess it really does not matter since it will get removed anyway